posts/app.py

```diff
@@ -83,6 +99,14 @@ def get_all_posts_for_event(event_id):
     return serialize_posts_to_json(post_list)
 
 
+def delete_post(post_id, author_id, collection):
+    """Deletes the post matching post_id and author_id if it exists."""
+    result = collection.delete_one(
+        {"_id": ObjectId(post_id), "author_id": author_id})
+    return (("Document deleted.", 204) if result.deleted_count
+            else ("Document not found.", 404))
```

Comment:
Do you want to return a 403 if the `_id` matches but the `author_id` does not?

Reply:
I was thinking about that, but the query and response are just a lot simpler this way instead of handling 3 cases.